src/Catalyst.Abstractions/Dfs/RepositoryData.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Catalyst.Abstractions.Dfs.CoreApi;
+
+namespace Catalyst.Abstractions.Dfs
+{
+    /// <summary>
+    ///   The statistics for <see cref="IStatsApi.RepositoryAsync"/>.
+    /// </summary>
+    public class RepositoryData
+    {
+        /// <summary>
+        ///   The number of blocks in the repository.
+        /// </summary>
+        /// <value>
+        ///   The number of blocks in the <see cref="IBlockRepositoryApi">repository</see>.
+        /// </value>
+        public ulong NumObjects;
+
+        /// <summary>
+        ///   The total number bytes in the repository.
+        /// </summary>
+        /// <value>
+        ///   The total number bytes in the <see cref="IBlockRepositoryApi">repository</see>.
+        /// </value>
+        public ulong RepoSize;
+
+        /// <summary>
+        ///   The fully qualified path to the repository.
+        /// </summary>
+        /// <value>
+        ///   The directory of the <see cref="IBlockRepositoryApi">repository</see>.
+        /// </value>
+        public string RepoPath;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4612924)